master/docs/manual/cfg-workers-openstack.rst

```diff
@@ -152,6 +152,28 @@ When the instance is destroyed, the volume will be terminated as well.
     ]
 
 
+The ``nova_args`` can be used to specify additional arguments for the novaclient. For example network mappings,
```

Comment:
we have the policy of maintaining one line per sentence in the doc (for new doc, and we try to slowly transition for old doc)